### PR TITLE
Updating docs and dependencies

### PR DESCRIPTION
**Pull request by kodedninja**
Fixing some errors I've experienced while installing Hashbase on a fresh Ubuntu droplet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,6 +73,7 @@
     "request": "^2.85.0",
     "request-promise-native": "^1.0.5",
     "rimraf": "^2.6.2",
+    "sqlite3": "^4.0.4",
```
Comment (pfrazee) [inline package.json]: Why did sqlite3 get added?

File: readme.md [hunk 2]
```diff
@@ -9,6 +9,15 @@ Links:
 
 ## Setup
 
+Make sure you have the following dependencies installed:
```
Comment (pfrazee) [inline readme.md]: Let's change this to 

```suggestion
On some platforms, you will need to have the following dependencies installed:
```

File: package.json (hunk 1 above)
Comment (kodedninja) [reply package.json]: It was required for some bindings.

Comment (pfrazee) [reply package.json]: Hm okay

**Pull request merged**